akka-http-core/src/main/java/akka/http/javadsl/model/HttpMethod.java

```diff
@@ -42,6 +42,14 @@ public final String name() {
 
     /**
      * Returns the entity acceptance level for this method.
+     * @deprecated Use {@link #getRequestEntityAcceptance} instead, which returns {@link akka.http.javadsl.model.RequestEntityAcceptance}.
      */
+    @Deprecated
```

Comment:
Note to self... Isn't this a bad idea, since ScalaDSL extends it, and then then the deprecation would leak through, but there it's correct actually.